### DISCUSSION TITLE
when under "advanced" for undervolting, add mV as the unit for the slider.

### DISCUSSION THREAD

**Discussion by ipiegat** [seerge/g-helper#3406] [Ideas]
change "-##" to "-##mV" in "advanced - undervolting" for clarity. like how power has "90W" and not "90". 

as a person initially new to undervolting, adding the unit would've given more clarity on what i was doing. plus it will match the other UI as an OCD freak.

**Comment by ipiegat** [comment]
you know the softwares good when this is my only critique lol

**Comment by envyska** [comment]
This guy doesnt know how amd curve optimizer works :P

**Comment by CodeCraftsman-Jr** [comment]
## Session Summary - 2026-04-18

**Completed**
- Implemented issue #49 core code changes for organization Drug License verification flow.
- Added backend endpoint /api/organization/onboarding/drug-license-verify and persistence in erification.drugLicense.
- Updated website onboarding Step 1 and Step 4 to use/read Drug License verification.
- Local validations passed: API syntax checks and website production build.

**In Progress**
- Staging smoke validation (stagingweb + stagingapi) before closing #49.

**Tracked**
- #49 - Replace Organization Registration Verification with SurePass Drug License

**Build Status**
- Blumber-Website 
pm run build passed

**Next Session Priority**
1. Run staging smoke tests for onboarding flow
2. Close #49 with staging validation evidence
